pkg/cli/cli.go

```diff
@@ -491,15 +492,31 @@ func cvdCommands(opts *subCommandOpts) []*cobra.Command {
 	// Local image
 	create.Flags().BoolVar(&createFlags.LocalImage, localImageFlag, false,
 		"Builds a CVD with image files built locally, the required files are https://cs.android.com/android/platform/superproject/+/master:device/google/cuttlefish/required_images and cvd-host-packages.tar.gz")
-	localImgMutuallyExFlags := []string{
+	// Local artifacts flag
+	d := "Path to local artifacts config file in toml format. For example:" +
+		"\n`````" + `
+Bootloader="PATH/bootloader"
+
+CVDHostPackageTar="PATH/cvd-host_package.tar.gz"
+
+[Images]
+boot = "PATH/boot.img"
+init_boot = "PATH/init_boot.img"
+super = "PATH/super.img"
+userdata = "PATH/userdata.img"
+` + "```"
+	create.Flags().StringVar(&createFlags.LocalArtifacts, localArtifactsFlag, "", d)
+	localBuildMutuallyExFlags := []string{
 		branchFlag, buildIDFlag, buildTargetFlag,
 		kernelBranchFlag, kernelBuildIDFlag, kernelBuildTargetFlag,
 		bootloaderBranchFlag, bootloaderBuildIDFlag, bootloaderBuildTargetFlag,
 		systemImgBranchFlag, systemImgBuildIDFlag, systemImgBuildTargetFlag,
 	}
-	for _, f := range localImgMutuallyExFlags {
+	for _, f := range localBuildMutuallyExFlags {
 		create.MarkFlagsMutuallyExclusive(f, localImageFlag)
+		create.MarkFlagsMutuallyExclusive(f, localArtifactsFlag)
```

Comment:
Shouldn't these two be mutually exclusive too?

The only reason I can image these not being mutually exclusive is using a combination of locally built host tools with downloaded/built-elsewhere guest, but that same argument applies to locally built host with guest from build api, so nothing would be mutually exclusive then.

Reply:
yes, these flags  are mutually exclusive, we had line `create.MarkFlagsMutuallyExclusive(localImageFlag, localArtifactsFlag)` down below, the code changed after using flags, however the mutual exclusion part stays.